manifests/crd/acceleratorprofiles.opendatahub.io.crd.yaml

```
@@ -28,27 +28,36 @@ spec:
               properties:
                 displayName:
                   type: string
+                  description: "The display name of the accelerator profile visible in the dashboard UI."
```

Is it necessary to qualify this description with `visible in the dashboard UI`?
```suggestion
                  description: "The display name of the accelerator profile."
```